tests/conftest.py

```diff
@@ -43,14 +43,12 @@ def setup_sample_data() -> Any:
 
         examples.load_css_templates()
         examples.load_world_bank_health_n_pop(sample=True)
-        examples.load_birth_names(sample=True)
 
     yield
 
     with app.app_context():
         engine = get_example_database().get_sqla_engine()
         engine.execute("DROP TABLE wb_health_population")
```

Comment:
Should we use `IF EXISTS` here also?

Reply:
Do you mean leaving sth like this: `engine.execute("DROP TABLE IF EXISTS birth_names")`?